dev-python/wxpython/wxpython-4.0.7-r2.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{8..10} )
+WX_GTK_VER="3.0-gtk3"
+
+DISTUTILS_IN_SOURCE_BUILD="1"
+
+inherit distutils-r1 multiprocessing virtualx wxwidgets
+
+MY_PN="wxPython"
+MY_PV="${PV}.post2"
+
+DESCRIPTION="A blending of the wxWindows C++ class library with Python"
+HOMEPAGE="https://www.wxpython.org/"
+SRC_URI="mirror://pypi/${P:0:1}/${MY_PN}/${MY_PN}-${MY_PV}.tar.gz"
+
+LICENSE="wxWinLL-3"
+SLOT="4.0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="test webkit"
+# Tests broken: #726812, #722716
+RESTRICT="test"
+#RESTRICT="!test? ( test )"
+
+# wxPython doesn't seem to be able to optionally disable features. webkit is
+# optionally patched out because it's so huge, but other elements are not,
+# which makes us have to require all features from wxGTK
+RDEPEND="
+	>=x11-libs/wxGTK-3.0.4-r301:${WX_GTK_VER}=[gstreamer,libnotify,opengl,sdl,tiff,webkit?,X]
+	media-libs/libpng:0=
+	media-libs/tiff:0
+	virtual/jpeg:0"
```

Comment:
Could you also update this deprecated dependency please?

Reply:
done